src/packets.c

```diff
@@ -946,7 +946,7 @@ TbBool process_dungeon_control_packet_clicks(long plyr_idx)
         if (player->work_state == PSt_CtrlDirect)
             thing = get_creature_near_for_controlling(plyr_idx, x, y);
         else
-            thing = get_creature_near_for_controlling(plyr_idx, x, y); // TODO Change to allow controlling any owner
+            thing = get_creature_near(x, y); // TODO Change to allow controlling any owner
```

Comment:
Didn't you already manage to implement this TODO statement? If so, the comment can be removed.